index.html

```diff
@@ -68,7 +68,78 @@ <h3>Multi-Post Stories</h3>
                 <div class="item3">bootstrap</div>
                 <div class="item3">Ruby</div>   
             </div>
-            <button>Select Project</button>
+            <button type="button" class="work-button">Select Project</button>
+            <div class="job1">
+                <h2>Profesional Art Printing Data</h2>
+                <div class="info-data"><p>A daily section of daily private personalised reads; no accounts or sign ups required ha been the industry standups</p></div>
+                <div class="grid-container2">
+                    <div class="item2">html</div>
+                    <div class="item3">bootstrap</div>
+                    <div class="item3">Ruby</div>   
+                </div>
+                <button type="button" class="job-button">Select Project</button>
+            </div>
+            <div class="job1">
+                <h2>Profesional Art Printing Data</h2>
+                <div class="info-data"><p>A daily section of daily private personalised reads; no accounts or sign ups required ha been the industry standups</p></div>
+                <div class="grid-container2">
+                    <div class="item2">html</div>
+                    <div class="item3">bootstrap</div>
+                    <div class="item3">Ruby</div>   
+                </div>
+                <button type="button" class="job-button">Select Project</button>
+            </div>
+            <div class="job1">
+                <h2>Profesional Art Printing Data</h2>
+                <div class="info-data"><p>A daily section of daily private personalised reads; no accounts or sign ups required ha been the industry standups</p></div>
+                <div class="grid-container2">
+                    <div class="item2">html</div>
+                    <div class="item3">bootstrap</div>
+                    <div class="item3">Ruby</div>   
+                </div>
+                <button type="button" class="job-button">Select Project</button>
+            </div>
```

Comment:
- kindly you must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). instead of `Select Project` change to `See Project`
- kindly follow the project requirement also in this section. you're required to use `ul` and `li`  for categories in "Work" section